src/lcmap/aardvark/tile_spec.clj

```diff
@@ -53,11 +53,11 @@
                             (hayt/where params)
                             (hayt/allow-filtering)))))
 
-(def column-names [:name :ubid :wkt :satellite :instrument
+(def column-names [:name :ubid :tags :wkt :satellite :instrument
                    :tile_x :tile_y :pixel_x :pixel_y :shift_x :shift_y
                    :band_product :band_category :band_name :band_long_name :band_short_name :band_spectrum
                    :data_fill :data_range :data_scale :data_type
-                   :data_units :data_shape #_:data_mask])
+                   :data_units :data_shape :data_mask])
```

Comment:
This was disabled and deserved a comment. It was disabled because saving a data_mask converted from JSON requires conversion of string keys to int keys. However, it seems unwise to support saving tile-specs via the REST API, so including a data_mask when saving tile-specs makes sense now.